### ISSUE TITLE
Constructor's polymorphism bug

### ISSUE THREAD

**Issue by RouxRC** [jacomyal/sigma.js#525]
The settings of the sigma instances are not taken into account by the renderer when instantiating sigma with container as first argument instead of inside a first object argument:
for instance:

```javascript
s = new sigma(
  'container',
  settings: {
    drawEdges: false
  });
```

displays edges, while:

```javascript
s = new sigma({
  container: 'container',
  settings: {
    drawEdges: false
  });
```

does not.

**Comment by Yomguithereal** [comment]
@RouxRC, can you remind me whether this polymorphism is expected and/or documented?

**Comment by RouxRC** [comment]
nope, my bad, I guess I typed it from adapting examples but it does not seem like an official way to instantiate sigma, so you can close and forget about it if you like, although passing the container as first argument and settings as second would be my preferred choice
